features/src/docker-out/installer.go

```diff
@@ -136,6 +142,37 @@ func (c *dockerCliComponent) InstallVersion(version *gover.Version) error {
 	if err := execr.Run(true, "cp", "-prf", "docker-init.sh", "/usr/local/share/docker-init.sh"); err != nil {
 		return err
 	}
+	// Copy the default config.json
+	if c.ConfigPath != "" {
+		fileContent, err := installer.ReadConfigFile(c.ConfigPath)
+		if err != nil {
+			return err
+		}
+		userName := os.Getenv("_REMOTE_USER")
```

Comment:
There should also exist a `_REMOTE_USER_HOME` variable that contains the users-home path, not sure if this is always the case and if it would be better to use it thou.